wgengine/magicsock/magicsock.go

```diff
@@ -2743,6 +2730,8 @@ func (c *RebindingUDPConn) Reset(pconn net.PacketConn) {
 	}
 }
 
+// ReadFromNetaddr reads a packet from c into b.
+// It returns the number of bytes copied and the return address.
```

Comment:
"the source address" I think is more common wording

Reply:
Done.